### ISSUE TITLE
feature request: delete button next to number of cookies per site

### ISSUE THREAD

**Issue by gthb1** [ysard/cookie-quick-manager#113]
Hi.

As deleting cookies per right click menu is slow and error-prone when focus jumps away,
please allow me to suggest the following feature:
"delete button next to number of cookies per site"
This would make it much easier to delete all cookies of a site.

Ideally next to that would be a "delete button next to number of cookies cookies per site" for the whole domain (as in example.com), so to avoid having to look for all the sneaky variations many sites "invent".

Even better would be a "delete button next to number of cookies per site" for all the domain as well as all cookies that domain's other functions did set (even if those are from other sites/domains, e.g. chat bots, help desks, extra 3rd party trackers, etc).

Thx.

**Comment by jpezz** [comment]
I agree. Some of these sites put cookies under so many names, it's hard to see which should be deleted. Particularly true with Google these days where they put youtube cookies along with Google cookies when doing some things.
